recipes/pyinterval/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "pyinterval" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8c46224a05815affa803ed5620432236bba620a246701b860d98ae7358a41d21
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Can this be noarch? It's okay for the Windows to CI to then fail as it will only be built on Linux
```suggestion
  noarch: python
```

---

Regarding the failing windows build, the upstream package supposedly supports building on windows, but it failed in the CI, and I have no ways of fixing that, thats why I added the skip there.
Regarding the `noarch`, I'm not sure of the definition of "pure python package" here, but my guess was this was not considered a pure python package, given its dependence on a c library.

---

I've only glanced at it so correct me if I'm wrong but `pyinterval` is only written in Python and depends on `crlibm` which is a mix of Python and C? From a packaging perspective we don't care how the dependencies are implemented, only if the library itself needs to be built differently on different platforms.

---

Oops, for this recipe I actually agree, I thought we were still discussing crlibm ...